models/base_model.py

```diff
@@ -461,6 +465,25 @@ def set_input_semantic_mask(self, data):
                     self.opt.data_online_context_pixels : -self.opt.data_online_context_pixels,
                 ]
 
+    def set_input_semantic_prompt(self, data):
```

Comment:
Rename to `set_input_prompt` as this function is not related to semantics.

Reply:
c'est fait